.github/workflows/build.yml

```diff
@@ -13,11 +13,16 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-    
+
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python 3.8
       uses: actions/setup-python@v2
       with:
         python-version: 3.8
     - uses: pre-commit/action@v2.0.0
+    - name: Run Tests
+      run: |
+        pip install --upgrade pip
+        pip install -e ".[dev]"
+        pytest -vv tests/tests.py
```

Comment:
probably just want to do tests/ so it'll pick up any other tests we add without changing anything. We'd also need to call tests.py something like test_helpers.py (see https://docs.pytest.org/en/latest/example/pythoncollection.html)